apps/ui/src/views/Proposal.vue

```diff
@@ -202,7 +202,7 @@ watchEffect(() => {
       </div>
       <Affix
         :class="[
-          'shrink-0 md:w-[340px] border-l-0 md:border-l',
+          'shrink-0 md:w-[340px] border-l-0 md:border-l -mb-6',
```

Comment:
This is adding additional space on votes page
<img width="1156" alt="Untitled 5" src="https://github.com/user-attachments/assets/550b97ce-a99c-4ade-9651-88be924f741b">

- Go to http://localhost:8080/#/s:starknet.eth/proposal/0xda9f5228db4cf70d057474e2e0a644cf626925a2da5aa4b5fa5317fb739eb7cb/votes
- Scroll to bottom